lib/glimesh/chat.ex

```diff
@@ -34,15 +34,23 @@ defmodule Glimesh.Chat do
   def create_chat_message(%User{} = user, %Channel{} = channel, attrs \\ %{}) do
     with :ok <- Bodyguard.permit(__MODULE__, :create_chat_message, user, channel) do
       if allow_link_in_message(channel, attrs) do
+        platform_subscriber = Glimesh.Payments.is_platform_subscriber?(user)
```

Comment:
I think this may be the wrong field. Platform subscriber is for Glimesh.tv directly, and is not specific to any one chat message. Channel subscriber is for the individual channel, and the users get a Sub icon when they have an active subscription.

Reply:
Sounds good! I'll remove this and retest any further assumptions i had here. 
This looks like a good candidate for reading this data in the user session, rather than the chat message function